### PR TITLE
feat(i18n): Simplify fetching custom resources

### PR DESCRIPTION
**Pull request by zhenlei520**
# Description

 Simplify fetching custom resources

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

File: src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomerResource.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.Ddd.Domain.Tests;
+
+public class CustomerResource
```
Comment (doddgu) [inline src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomerResource.cs:6]: custom？customer？

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomerResource.cs (hunk 1 above)
Comment (zhenlei520) [reply src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomerResource.cs]: Processed

File: src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomResource.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 namespace Masa.BuildingBlocks.Ddd.Domain.Tests;
 
-public class CustomerResource
+public class CustomResource
```
Comment (doddgu) [inline src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomResource.cs:6]: file name

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomResource.cs (hunk 2 above)
Comment (zhenlei520) [reply src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomResource.cs:6]: The file name has been changed, you may need to refresh it

**Review by doddgu — APPROVED** [review]

**Pull request merged**